### PR TITLE
[Style/#89]:구매자 관리 페이지 좌측 정보 UI

### PR DESCRIPTION
**Pull request by juhee067**
## 📌 작업 내용

- 구매자 관리 페이지 좌측 정보 UI
- 페이지별 headerInfo 추가
- ProductCard size 속성 추가
- ManagementLayout 
```jsx
function ManagementPage() {
  return (
    <div>
      <ManagementLayout>
        <BuyDetail />
      </ManagementLayout>
    </div>
  );
}
```
```jsx
function ManagementLayout({ children }: Props) {
  const location = useLocation();
  const { id } = useParams();

  let headText: string | undefined = `총 ${1500}병 신청됐습니다`;
  let buttonProps: ButtonProps = {
    label: '확정',
    $style: 'solid',
    $theme: 'primary',
  };
  if (id) {
    headText = undefined;
  }
  if (location.search === '?achievement') {
    buttonProps = {
      label: '전체확정',
      $style: 'outlined',
      $theme: 'primary',
    };
  } else if (location.search === '?shortfall') {
    buttonProps = {
      label: '미달성',
      $style: 'outlined',
      $theme: 'neutral',
    };
  } else if (location.search === '?cancel') {
    buttonProps = {
      label: '전체경고',
      $style: 'solid',
      $theme: 'primary',
    };
    headText = undefined;
  } 

  return (
    <Container>
      <ProductCard
        liquorInfo={liquors}
        headText={headText}
        button={buttonProps}
      />
      {children}
    </Container>
  );
}

export default ManagementLayout;
```
<br/>

## 📌 구현 결과 (선택)
image
<br/>

## 📌 기타 사항

- router 페이지에서 감싸지않고 page내부에서 감쌌는데 검색창을 포함해서 layout을 짜고 때문에 그냥 router페이지에서 감싸는 게 맞는 거 같단 생각이 듭니다.
- 그리고 탭별, 구매자 상세보기에서 달라지는 버튼 스타일을 저런식으로 코드를 짜는 게 너무 지저분해서 맞는지 의문이 듭니다.

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">8.73%</td> <td align="right">545 / 6237</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">8.73%</td> <td align="right">545 / 6237</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">16.53%</td> <td align="right">43 / 260</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">26.35%</td> <td align="right">78 / 296</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx">src/features/Catalog/components/Main/ProductCard.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L5-L6" class="text-red">5-6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L28-L29" class="text-red">28-29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L31-L35" class="text-red">31-35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L37-L44" class="text-red">37-44</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L46-L48" class="text-red">46-48</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L50" class="text-red">50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L52-L62" class="text-red">52-62</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L64" class="text-red">64</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L66" class="text-red">66</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L68-L70" class="text-red">68-70</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L73" class="text-red">73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L80" class="text-red">80</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L98" class="text-red">98</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L105-L106" class="text-red">105-106</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L112" class="text-red">112</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L116" class="text-red">116</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L121" class="text-red">121</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L125" class="text-red">125</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L134-L135" class="text-red">134-135</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L138" class="text-red">138</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Catalog/components/Main/ProductCard.tsx#L141" class="text-red">141</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx">src/features/LiquorDetail/components/Fallback.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L3" class="text-red">3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L5-L8" class="text-red">5-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L10-L11" class="text-red">10-11</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L13" class="text-red">13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L15" class="text-red">15</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/Fallback.tsx#L23-L24" class="text-red">23-24</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx">src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L5" class="text-red">5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L11-L13" class="text-red">11-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L15-L31" class="text-red">15-31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L33" class="text-red">33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L35" class="text-red">35</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L37" class="text-red">37</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L49-L50" class="text-red">49-50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L55" class="text-red">55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L71-L72" class="text-red">71-72</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L76-L77" class="text-red">76-77</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L82" class="text-red">82</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/LiquorDetail/components/ReviewBox/ReviewListItem.tsx#L91-L92" class="text-red">91-92</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx">src/features/Management/components/BuyerDetail/BuyDetail.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L1-L7" class="text-red">1-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L9-L24" class="text-red">9-24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L26-L27" class="text-red">26-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L29" class="text-red">29</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L31" class="text-red">31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyDetail.tsx#L33" class="text-red">33</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx">src/features/Management/components/BuyerDetail/BuyerTable.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L1-L3" class="text-red">1-3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L6-L12" class="text-red">6-12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L14-L33" class="text-red">14-33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L35-L47" class="text-red">35-47</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L49-L55" class="text-red">49-55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L57" class="text-red">57</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L59" class="text-red">59</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L61" class="text-red">61</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L71" class="text-red">71</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L76-L77" class="text-red">76-77</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L83" class="text-red">83</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L88-L89" class="text-red">88-89</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L96-L97" class="text-red">96-97</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L99" class="text-red">99</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/features/Management/components/BuyerDetail/BuyerTable.tsx#L103-L104" class="text-red">103-104</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx">src/layouts/ManagementLayout.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L1" class="text-red">1</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L3" class="text-red">3</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L6-L7" class="text-red">6-7</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L9-L14" class="text-red">9-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L20-L21" class="text-red">20-21</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L23-L27" class="text-red">23-27</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L29-L55" class="text-red">29-55</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L57-L65" class="text-red">57-65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L67" class="text-red">67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L69" class="text-red">69</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/layouts/ManagementLayout.tsx#L71" class="text-red">71</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/pages/BuyerDetailPage.tsx">src/pages/BuyerDetailPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/pages/BuyerDetailPage.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/pages/BuyerDetailPage.tsx#L4-L10" class="text-red">4-10</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/pages/BuyerDetailPage.tsx#L12" class="text-red">12</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/66b7d8889ad3dc70630867a69a910f4333818fd2/src/pages/BuyerDetailPage.tsx#L14" class="text-red">14</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10524580138>#151</a></em>

File: src/layouts/ManagementLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  let headText: string | undefined = `총 ${1500}병 신청됐습니다`;
+  let buttonProps: ButtonProps = {
+    label: '확정',
+    $style: 'solid',
+    $theme: 'primary',
+  };
+  if (id) {
+    headText = undefined;
+  }
+  if (location.search === '?achievement') {
```
Comment (JIMIN1020) [inline src/layouts/ManagementLayout.tsx:33]: location.search를 변수로 만들면 되지 않을까요?

**Review by JIMIN1020 — APPROVED** [review]
고생하셨습니다 주희님~~~!

File: src/layouts/ManagementLayout.tsx (hunk 1 above)
Comment (juhee067) [reply src/layouts/ManagementLayout.tsx:33]: 알겠습니다!

File: src/layouts/ManagementLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+import React from 'react';
+import { useLocation, useParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+const liquors = {
+  img: MockImage,
+  categoryName: '마감',
+  name: '홉미드',
+  description: '고세종고세종고세종고세종고세종',
```
Comment (SebellKo) [inline src/layouts/ManagementLayout.tsx:13]: ㅋㅋ 심술 부리시니 ci 에러 나시잖아요 주히님 ~

**Review by SebellKo — APPROVED** [review]
주히님 고생고생 하셨습니다 ~

File: src/layouts/ManagementLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  const { id } = useParams();
+
+  let headText: string | undefined = `총 ${1500}병 신청됐습니다`;
+  let buttonProps: ButtonProps = {
+    label: '확정',
+    $style: 'solid',
+    $theme: 'primary',
+  };
+
+const src =location.search;
```
Comment (ksy90101) [inline src/layouts/ManagementLayout.tsx:31]: CI lint

File: src/layouts/ManagementLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (ksy90101) [inline src/layouts/ManagementLayout.tsx:32]: CI lint

File: src/layouts/ManagementLayout.tsx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  
+  if (id) {
+    headText = undefined;
+  }
+
```
Comment (ksy90101) [inline src/layouts/ManagementLayout.tsx:37]: CI lint

File: src/layouts/ManagementLayout.tsx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+      $theme: 'neutral',
+    };
+  } else if (src === '?cancel') {
+    buttonProps = {
+      label: '전체경고',
+      $style: 'solid',
+      $theme: 'primary',
+    };
+    headText = undefined;
+  }
```
Comment (ksy90101) [inline src/layouts/ManagementLayout.tsx:57]: 이런 경우 객체로 만들어서 많이 처리하긴 하는데.
if문이 복잡성이 높아지니깐요.

**Review by ksy90101 — COMMENTED** [review]
CI Lint . 잘확인해주세요.

**Pull request merged**